luigi_pipeline/seqr_loading.py

```diff
@@ -11,12 +11,42 @@
 from lib.model.seqr_mt_schema import SeqrVariantSchema, SeqrGenotypesSchema, SeqrVariantsAndGenotypesSchema
 
 logger = logging.getLogger(__name__)
-
+GRCh37_STANDARD_CONTIGS = {'1','10','11','12','13','14','15','16','17','18','19','2','20','21','22','3','4','5','6','7','8','9','X','Y', 'MT'}
+GRCh38_STANDARD_CONTIGS = {'chr1','chr10','chr11','chr12','chr13','chr14','chr15','chr16','chr17','chr18','chr19','chr2','chr20','chr21','chr22','chr3','chr4','chr5','chr6','chr7','chr8','chr9','chrX','chrY', 'chrM'}
+OPTIONAL_CHROMOSOMES = ['MT', 'chrM', 'Y', 'chrY']
+VARIANT_THRESHOLD = 100
 
 def check_if_path_exists(path, label=""):
     if (path.startswith("gs://") and not hl.hadoop_exists(path)) or (not path.startswith("gs://") and not os.path.exists(path)):
         raise ValueError(f"{label} path not found: {path}")
 
+def contig_check(mt, standard_contigs, threshold):
+    row_dict = mt.aggregate_rows(hl.agg.counter(mt.locus.contig))
+    contigs_set = set(row_dict.keys())
+    # chromosomes that are not in the VCF
+    missing_contigs = standard_contigs - contigs_set
+    result_dict = {contig: None for contig in missing_contigs}
```

Comment:
I'm not really sure why you are merging all the different errors into a single result dictionary and then parsing out the values to figure out what the error was just so you can log it. It would be much clearer if you log errors as they happen and keep the errors in whatever format you plan to return them from the function, as nothing you see later on is going to change the error that already happened. For example, here you would do something like
```
if missing_contigs:
    logger.warning('Missing the following chromosome(s): {}'.format(', '.join(missing_contigs))
```